src/themes/default/globals/colors.less

```diff
@@ -3,617 +3,617 @@
 
-    transparentBackground: @transparentFormSuccessBackground;
-    transparentColor: @transparentFormSuccessColor;
-  };
+        transparentBackground: @transparentFormSuccessBackground;
+        transparentColor: @transparentFormSuccessColor;
+    };
 
-  @warning: {
-    color: @formWarningColor;
-    background: @formWarningBackground;
-    borderColor: @formWarningBorder;
-    borderRadius: @inputWarningBorderRadius;
-    boxShadow: @inputWarningBoxShadow;
-    cornerLabelColor: @white;
-    labelBackground: @formWarningLabelBackground;
+    @warning: { // stylelint-disable-line indentation
+        color: @formWarningColor;
+        background: @formWarningBackground;
+        borderColor: @formWarningBorder;
+        borderRadius: @inputWarningBorderRadius;
+        boxShadow: @inputWarningBoxShadow;
+        cornerLabelColor: @white;
+        labelBackground: @formWarningLabelBackground;
 
-    dropdownLabelColor: @dropdownWarningLabelColor;
-    dropdownLabelBackground: @dropdownWarningLabelBackground;
-    dropdownHoverBackground: @dropdownWarningHoverBackground;
-    dropdownSelectedBackground: @dropdownWarningSelectedBackground;
-    dropdownActiveBackground: @dropdownWarningActiveBackground;
+        dropdownLabelColor: @dropdownWarningLabelColor;
+        dropdownLabelBackground: @dropdownWarningLabelBackground;
+        dropdownHoverBackground: @dropdownWarningHoverBackground;
+        dropdownSelectedBackground: @dropdownWarningSelectedBackground;
+        dropdownActiveBackground: @dropdownWarningActiveBackground;
 
-    inputAutoFillBackground: @inputAutoFillWarningBackground;
-    inputAutoFillBorderColor: @inputAutoFillWarningBorder;
-    inputFocusBackground: @inputWarningFocusBackground;
-    inputFocusColor: @inputWarningFocusColor;
-    inputFocusBorderColor: @inputWarningFocusBorder;
-    inputFocusBoxShadow: @inputWarningFocusBoxShadow;
-    inputPlaceholderColor: @inputWarningPlaceholderColor;
-    inputPlaceholderFocusColor: @inputWarningPlaceholderFocusColor;
+        inputAutoFillBackground: @inputAutoFillWarningBackground;
+        inputAutoFillBorderColor: @inputAutoFillWarningBorder;
+        inputFocusBackground: @inputWarningFocusBackground;
+        inputFocusColor: @inputWarningFocusColor;
+        inputFocusBorderColor: @inputWarningFocusBorder;
+        inputFocusBoxShadow: @inputWarningFocusBoxShadow;
+        inputPlaceholderColor: @inputWarningPlaceholderColor;
+        inputPlaceholderFocusColor: @inputWarningPlaceholderFocusColor;
 
-    transparentBackground: @transparentFormWarningBackground;
-    transparentColor: @transparentFormWarningColor;
-  };
+        transparentBackground: @transparentFormWarningBackground;
+        transparentColor: @transparentFormWarningColor;
+    };
 };
 
-@textStates: {
-  @error: {
-    color: @negativeColor;
-  };
-  @info: {
-    color: @infoColor;
-  };
-  @success: {
-    color: @positiveColor;
-  };
-  @warning: {
-    color: @warningColor;
-  };
-};
```

Comment:
Don't remove the semicolon here, it was intentially added by #1833 to support a buggy PHP LESS mentioned in #1832

Reply:
fixed in https://github.com/fomantic/Fomantic-UI/pull/2592/commits/6c88db28178de841f3fec3eb1e20a511670a3e1c